core/src/main/java/io/kestra/plugin/core/flow/ForEach.java

```diff
@@ -89,24 +90,24 @@
                 so the `return` task will run concurrently for the first two values in the list at first. \
                 The `return` task will run for the next two values only after the task runs for the first two values \
                 have completed.""",
-            code = {
-                "id: for_each_value",
-                "namespace: company.team",
-                "",
-                "tasks:",
-                "  - id: for_each",
-                "    type: io.kestra.plugin.core.flow.ForEach",
-                "    values: ",
-                "      - value 1",
-                "      - value 2",
-                "      - value 3",
-                "      - value 4",
-                "    concurrencyLimit: 2",
-                "    tasks:",
-                "      - id: return",
-                "        type: io.kestra.plugin.core.debug.Return",
-                "        format: \"{{ task.id }} with value '{{ taskrun.value }}'\"",
-            }
+            code = """
+                id: for_each_value
+                namespace: company.team
+                
+                tasks:
+                  - id: for_each
+                    type: io.kestra.plugin.core.flow.ForEach
+                    values: 
+                      - value 1
+                      - value 2
+                      - value 3
+                      - value 4
+                    concurrencyLimit: 2
+                    tasks:
+                      - id: return
+                        type: io.kestra.plugin.core.debug.Return
+                        format: "{{ task.id }} with value '{{ taskrun.value }}'"
```

Comment:
Does `{{ taskrun.value }}` need to be wrapped in additional single quotes?

Reply:
Not required.. removing.